iceoryx_hoofs/testing/CMakeLists.txt

```diff
@@ -38,6 +37,8 @@ iox_add_library(
         mocks/error_handler_mock.cpp
         timing_test.cpp
         compile_test.cpp
+    ${IOX_EXPORT_GTEST_LIBS} # ADDITIONAL_EXPORT_TARGETS Argument set on request
+    ${IOX_GTEST_GMOCK_INCLUDE} # PUBLIC_INCLUDES Argument set on request
```

Comment:
This looks suspicious. Either it does not belong here or the names are misleading.